<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Shape Recognition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GraphicsLayoutWidget" name="view0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="view1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="view2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="view5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="view4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="view3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="sensor_index">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>385</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="sensor_thumb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>563</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="initialize">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>General</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="configure">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="init_main">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_main">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toHome">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Move to &quot;Home&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveDown">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Move down</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UR10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="toBase">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Move to base</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveUp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Move up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toPose">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set initial pose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>iLimb</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pinch">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Feedback pinch touch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrate">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>560</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopSensors">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tactile Sensors</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startSensors">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>530</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualize">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>620</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize point cloud</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>600</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape Recognition</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="convexHull">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>650</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute convex hull</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ccw">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CCW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cw">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveAway">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Move away</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detectShape">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>680</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect shape</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>710</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear values</string>
    </property>
   </widget>
   <widget class="QComboBox" name="port_iLimb">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="port_tactile">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_ur10">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>47</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UR10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>85</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>iLimb</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>115</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tactile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>501</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>20</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
